include/wx/graphics.h

```diff
@@ -449,6 +431,95 @@ class WXDLLIMPEXP_CORE wxGraphicsGradientStops
     wxVector<wxGraphicsGradientStop> m_stops;
 };
 
+// ----------------------------------------------------------------------------
+// wxGraphicsPenInfo describes a wxGraphicsPen
+// ----------------------------------------------------------------------------
+
+class wxGraphicsPenInfo : public wxPenInfoBase<wxGraphicsPenInfo>
```

Comment:
Not sure if this really had to be moved? It looks more logical to me to have it before `wxGraphicsPen`, even though there is no actual dependency of the latter on the former.

Reply:
That was probably due to a first pass I did for this change that I eventually mostly undid. There was something where the forward declare wasn't sufficient, but I don't see what it might have been now so I'll move it back.

Reply:
Oh, I remember now. Adding the `wxGraphicsGradientStops` to `wxGraphicsPenInfo` is what caused the ordering conflict. I'll switch this back and then readjust for that dependency.